src/client/templates/room.html

```diff
@@ -29,9 +29,11 @@ <h1 class="text-info">Sprint Retrospective</h1>
                 </div>
             </div>
         </div>
-        <div class="form-group" style="margin-top: 75px">
+        {{> sharejsAce docid=this onRender=config id="editor"}}
+        <button type="button" id="keyNoteSaveButton" class="btn btn-success">Save Keynotes</button>
+        <!--<div class="form-group" style="margin-top: 75px">
             <label for="notes-textarea">Notes</label>
```

Comment:
remove this if its no longer needed/obsolete.  We can come back and look at github if we need it again for some reason.  No reason to clutter up the codebase with commented out obsolete code